middleware/lib/sfMiddleware.ts

```diff
@@ -167,7 +167,7 @@ function decodeCeAttrib(attrVal: string|undefined): any {
 * @param logger      -- Logger
 * @return returnArgs -- array of arguments that make-up the user functions arguments
 */
-export function applySfFnMiddleware(cloudEvent: CloudEvent, headers: ReadonlyMap<string,string>, logger: Logger): Array<any> {
+export function applySfFnMiddleware(cloudEvent: CloudEvent, headers: ReadonlyMap<string,string>, logger: Logger | any): Array<any> {
```

Comment:
Interested in why `Logger|any` is required in these signatures -- I'd be more comfortable if it was `Logger|undefined` -- what error did we hit that required this?

Reply:
The `any` was required because the `Logger` was being referenced from two different versions. Typescript was throwing errors because the 2 `Logger` definitions didn't match.